Tests/CredentialsFacebookTests/TestTypeSafeToken.swift

```diff
@@ -166,19 +193,58 @@ class TestTypeSafeToken : XCTestCase {
         guard let profileInstance1 = TestFacebookToken.decodeFacebookResponse(data: testFacebookResponse) else {
             return XCTFail("Facebook JSON response cannot be decoded to TestFacebookToken")
         }
-        guard let profileInstance2 = FacebookTokenProfile.decodeFacebookResponse(data: testFacebookResponse) else {
-            return XCTFail("Facebook JSON response cannot be decoded to FacebookTokenProfile")
+        guard let profileInstance2 = TestFacebookToken.decodeFacebookResponse(data: testFacebookResponse) else {
+            return XCTFail("Facebook JSON response cannot be decoded to TestFacebookToken")
```

Comment:
This was a mistake in the original test which I thought I'd fix while I was here.  The test is meant to insert two tokens into the same cache, and since the cache is associated with the type, the two tokens need to be instances of the same type.